apps/ledger-live-desktop/src/renderer/hooks/useActionCards.tsx

```diff
@@ -54,6 +54,11 @@ const useActionCards = () => {
     const actionCard = findActionCard(cardId);
 
     if (currentCard) {
+      // For some reason braze won't log the click event if the card url is empty
+      // Setting it as the card id just to have a dummy non empty value
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      currentCard.url = currentCard.id;
```

Comment:
Is it not dangerous to mutate the `currentCard` URL value? Don't we want to do it only for `logContentCardClick` function?

Reply:
Technically this property is only used for "classic" content cards but as we only leverage the custom design ones this property is never set but ideally yes we shouldn't mutate an object like this. I pushed a modification :)